### DISCUSSION TITLE
Adding New Users

### DISCUSSION THREAD

**Discussion by scottw12** [vanjarosoftware/Vanjaro.Platform#1948] [Q&A]
Problems with adding new users
We can add a new user and that users' credentials are accepted. But it seems they are not really logged in the sign-in block still show my account and not the login account name. That user is in the roles Registered Users and Editors, and we have set page permissions appropriately.  If I make them a superuser all works correctly as expected they have super user access. Not what I am after, they cannot access the Vanjaro menu on any page.

Has anyone else had this problem that can lead me in the right direction?

**Comment by meetmandeep** [comment]
> Problems with adding new users

How are you adding the user? Using the Sign Up Block or using the Site Menu (Admin)

**Comment by scottw12** [comment]
Site Menu
http://somup.com/crlnYu0x6t

**Comment by scottw12** [comment]
Well I moved our dev site to live and now the site looks great until you login and then it goes to hell ugg. Help would greatly be appreciated.

This also relates to issue #1941 I opened up 2 days ago.

Any help is appreciated I have an association board reviewing the site in the morning.

I’ll buy ya a beer of cup of coffee sometime. 😊

 http://somup.com/crlenF0xyV <--screen cap

https://Jcesba.org

 

Thanks

**Comment by scottw12** [comment]
I believe I found a workaround to my issues and deleted the last revisions of a couple of Global Blocks, from the dbo.VJ_Core_GlobalBlock table, and this stopped the issue on all pages.
How it became an issue to begin with I am not sure....  It seems to have happened when i was working in the global bock editing and drug an element outside of the block. Ya I am confused just as much as I have now made you confused lol
